ibtk/include/ibtk/libmesh_utilities.h

```diff
@@ -585,6 +585,25 @@ interpolate(libMesh::TypeVector<double>& U,
     return;
 } // interpolate
 
+/**
+ * Compute the jacobian with respect to the initial configuration in the deformed configuration
```

Comment:
We should add a formula defining the Jacobian here. I think we can do this with MathJax.

Reply:
I don't know how to use mathjax, but I'm willing to write out the equations